aiotarantool.py

```diff
@@ -320,17 +336,8 @@ def generate_sync(self):
             self.connected = False
             self._writer.transport.close()
             self._reader_task.cancel()
-            self._reader_task = None
 
             self._writer_task.cancel()
-            self._writer_task = None
```

Comment:
@shveenkov, I need your advice about that. I removed it because for some reason when connection is unstable and I try reconnecting, I am getting an error "None hasn't attribute read_eof". Could you clarify if it's alright?